MAUI/Cartesian-Charts/Axis/AutoScrollingDelta.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Auto scrolling delta in .NET MAUI Chart control | Syncfusion
+description: Learn here all about auto scrolling delta and its features in Syncfusion .NET MAUI Chart (SfCartesianChart) control and more.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# AutoScrollingDelta in .NET MAUI Chart
+
+[AutoScrollingDelta]() ensures that the specified range of data is always visible in the chart. It always shows the recently added data points at the end, and scrolling will be reset to the end of the range whenever a new point is added.
```

Comment:
The auto-scrolling of [SfCartesianChart]() ensures that the specified range of data is always visible in the chart. It always shows the recently added data points at the end, and scrolling will be reset to the end of the range whenever a new point is added. The [AutoScrollingDelta ]() property of the chart axis can be used to set the number of data points to be always visible in the chart.

Reply:
Replaced the above content